libs/application-generic/src/usecases/create-notification-jobs/create-notification-jobs.usecase.ts

```diff
@@ -88,6 +97,7 @@ export class CreateNotificationJobs {
       channels,
       controls: command.controls,
       tags: command.template.tags,
+      severity: isNotificationSeverityEnabled ? command.template.severity : undefined,
```

Comment:
the `notification` document will store the `severity` field and later we will pass it to the messages when we create them in the corresponding usecases